src/main/java/com/faforever/api/data/domain/Player.java

```diff
@@ -7,22 +7,26 @@
 import lombok.Setter;
 
 import javax.persistence.Entity;
+import javax.persistence.FetchType;
 import javax.persistence.OneToMany;
 import javax.persistence.OneToOne;
 import javax.persistence.Table;
 import javax.persistence.Transient;
 import java.util.List;
+import java.util.stream.Collectors;
 
 @Entity
 @Table(name = "login")
 @Include(rootLevel = true, type = "player")
-@SharePermission(any = Role.ALL.class) // Needed to change leader of a clan
+@SharePermission(any = Role.ALL.class) // Needed e.g. to change leader of a clan
 @Setter
 public class Player extends Login {
 
   private Ladder1v1Rating ladder1v1Rating;
   private GlobalRating globalRating;
   private List<ClanMembership> clanMemberships;
+  private List<BanInfo> bans;
+  private List<BanInfo> createdBans;
```

Comment:
Just wondering, will we ever need to know which bans a user created? That could get quite a big list. Just like we don't provide a list of hosted games per user, I'm not sure whether we want to provide this list :)